### ISSUE TITLE
Typehints in global namespace

### ISSUE THREAD

**Issue by daniel-sc** [goaop/framework#368]
It seems there is a bug with typehints in global namespace.
I documented this with the following commit https://github.com/daniel-sc/framework/commit/4439c4d445ad151560a0fc5dfa3ede0629c7edc5 on the branch https://github.com/daniel-sc/framework/tree/bug-typehint

The test fails, because the woven class contains the following problematic typehint:
```php
public function publicMethodFixedArguments(\\Exception $a, $b, $c = NULL)
```

This leads to errors in reality like:
```
ParseError: syntax error, unexpected '\' (T_NS_SEPARATOR), expecting identifier (T_STRING)
```

**Comment by lisachenko** [comment]
Hi, @daniel-sc!

Thank you for reporting this, looks like there should be ltrim here for normalizing global namespace.

**Comment by daniel-sc** [comment]
Hi @lisachenko 

I updated the PR accordingly.

**Comment by lisachenko** [comment]
Closed in #369
